server/sonar-web/Gruntfile.coffee

```diff
@@ -336,6 +342,14 @@ module.exports = (grunt) ->
           parallel: true
           port: expressPort
         src: ['<%= pkg.sources %>js/tests/e2e/tests/**/*.js']
+      testCoverage:
```

Comment:
@jblievremont Well, with `parallel: true` it randomly fails on tests with browser popup windows. I didn't find the reason yet.